code/controllers/subsystem/SSthrowing.dm

```diff
@@ -71,6 +71,8 @@ SUBSYSTEM_DEF(throwing)
 	///When this variable is false, non dense mobs will be hit by a thrown item. useful for things that you dont want to be cheesed by crawling, EG. gravitational anomalies
 	var/dodgeable = TRUE
 	/// Can a thrown mob move themselves to stop the throw?
+	var/should_block_movement = TRUE
+	/// This is internal variable to support no gravity movement even if variable above is TRUE. This variable will change during throwing
```

Comment:
`The variable above` isn't helpful for popups when the var is accessed elsewhere, please tweak it